staging-testnet.yaml

```diff
@@ -1,37 +1,34 @@
 args:
```

Comment:
I believe we should keep `staging-testnet.yaml` as is -- or even, rename it to `staging.yaml`.

The new file should be named `testnet.yaml`.

Yes, it was terribly confusing.. : -/